src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/FiltersForm.test.tsx

```diff
@@ -246,17 +286,47 @@ describe('FiltersForm', () => {
     ).toBeInTheDocument();
 
     expect(
-      screen.getAllByRole('checkbox', {
-        hidden: true,
-        name: (_, element) => (element as HTMLInputElement).checked,
-      }),
+      within(
+        screen.getByRole('group', {
+          name: 'School type',
+          hidden: true,
+        }),
+      ).getAllByRole('checkbox', { hidden: true }),
     ).toHaveLength(1);
 
     expect(screen.getByLabelText('State-funded secondary')).toHaveAttribute(
       'checked',
     );
   });
 
+  test('automatically select filter when there is only one', () => {
+    render(
+      <FiltersForm
+        {...testWizardStepProps}
+        subjectMeta={testSubjectMetaOneFilter}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(
+      screen.getByRole('button', {
+        name: 'Characteristic - 1 selected',
+      }),
+    ).toBeInTheDocument();
+
+    expect(
+      within(
+        screen.getByRole('group', {
+          name: 'Characteristic',
+        }),
+      ).getAllByRole('checkbox'),
+    ).toHaveLength(1);
+
+    expect(
+      screen.getByLabelText('Ethnicity Major Black Total'),
```

Comment:
Might want to check that this checkbox is specifically the single one within the filter group